index.html

```diff
@@ -2734,15 +2734,14 @@ <h2>
       </h2>
 
       <p>
-Section <a href="#method-operations"></a> specifies the <a>DID</a> operations to be
-supported by a <a>DID method</a> specification, including deactivation of a
-<a>DID document</a> by replacing it with an updated <a>DID document</a>. In
-general, checking for key revocation on <a>DLT</a>-based methods is expected to
-be handled in a manner similar to checking the balance of a cryptocurrency
-account on a <a>distributed ledger</a>. That is, if the balance is empty, the
-entire <a>DID</a> is deactivated. <a>DID method</a> specifications are expected
-to enable support for a quorum of trusted parties to enable key recovery. Some
-of the facilities to do so are suggested in Section
+Section <a href="#method-operations"></a> specifies the <a>DID</a> operations to
+be supported by a <a>DID method</a> specification, including deactivation of a
+<a>DID document</a> by replacing it with an updated <a>DID document</a>. As an
+example, removal of a cryptographic key from a <a>DID document</a> is an
+indication that the key has been revoked and is no longer valid. That is, if the
+balance is empty, the entire <a>DID</a> is deactivated. <a>DID method</a>
```

Comment:
There's still a bit about balance being empty here. I'm still not sure I follow the example in context, even with the update. Perhaps there's just something implicit that people who understand DLTs get that I'm missing though. I'd suggest removing this sentence altogether unless there's a way to make it clearer by making whatever is implicit explicit. Is it maybe that removing *all* keys is an indication that the entire DID is deactivated? (Not sure if this is true, more likely to be method-dependent because they won't all rely on key presence..)

```suggestion
<a>DID document</a> by replacing it with an updated <a>DID document</a>. <a>DID method</a>
```

Reply:
Nice catch. I missed that sentence that still had balance included in there. I'm good for removing an example all together because this is mentioned in section 7.3 already. My intention was to remove any reference to DLT specific examples and indicate a general example. I just pushed a commit with the example removed and indicating it's defined by the different did methods.